### DISCUSSION TITLE
Nanopi support

### DISCUSSION THREAD

**Discussion by sispeo29** [tesla-android/issue-tracker#225] [Q&A]
Hello

I may not have searched enough, but I don't find any information regarding possible other platform support.
Right now, it is difficult to buy Raspberry pi hardware. Any chance to make it work with other arm based cards ? (like nanopi for example) 

Thanks

**Comment by mikegapinski** [comment]
Hey!

Tesla Android and Android distributions in general are not universal. The only thing those other boards have in common with the raspberry pi is the form factor. There is no support for those clones planned, drivers for them are not easily available and I rely on rpi specific features heavily

**Reply by sispeo29** [reply]
Hello
Well they share a little bit more than the size of the board. They are both based on ARM processor. I was asking because it is very difficult to find one Rpi at a correct price at this time.
I am in mind that most of the difficulties relies on the OS support , but mabe I am wrong
Wondering if you made any kernel modification in order to make the things work on Rpi
